Automattic/jetpack
# DNM: First attempt at including Jetpack Backup UI into Jetpack

This is very dirty right now. Do not merge.

A first attempt at seeing what it would look like to include the Backup UI from the standalone into Jetpack proper. There are better ways of doing this (e.g. a package), but wanted to get a sense for what it would look like before extracting packages, etc.

#### Changes proposed in this Pull Request:

* Adds the Backup UI to Jetpack.

#### Jetpack product discussion


none yet.

#### Does this pull request change what data or activity we track or use?


no.

#### Testing instructions:






* Add `define( 'JETPACK__BACKUP_UI_BETA_ENABLED', true );` to your test site's wp-config.php with Jetpack active, without Backup active.
* See the new Backup menu on the left nav under Jetpack. Check it out.
